src/tools/interop/interop.cpp

```diff
@@ -739,6 +739,8 @@ RunInteropTest(
     Settings.IsSet.SendBufferingEnabled = TRUE;
     Settings.IdleTimeoutMs = WaitTimeoutMs;
     Settings.IsSet.IdleTimeoutMs = TRUE;
+    Settings.XdpEnabled = TRUE;
```

Comment:
wouldn't you need something like `if (Feature == XDP_enabled)` here?

Reply:
No. Features here are protocol features, so XDP isn't at that level.